docs/demos/themed/code.example

```diff
@@ -0,0 +1,62 @@
+const { Component } = React;
+
+const styles = {
+	alert: {
+		composes: "alert",
+		marginBottom: 6,
+		display: "inline-block",
+		textAlign: "left"
+	},
+	info: {
+		composes: "alert-info"
+	},
+	success: {
+        color: "#efd8f0",
+        background: "#8f2ba1",
+        borderColor: "#80237f",
+        borderWidth: 15,
+        borderRadius: 60
+	},
+	warning: {
+		composes: "alert-warning",
+	},
+	danger: {
+		composes: "alert-danger"
+	},
+	dismissable: {
+		composes: "alert-dismissable"
+	},
+	close: {
+		composes: "close"
+	},
+	msgContainer: {
+		display: "inline-block"
+	},
+	icon: {
+		verticalAlign: "top",
+		fontSize: 18,
+		paddingRight: 15,
+		opacity: 0.2
+	},
+	headline: {
+		margin: 0,
+		marginBottom: 6
+	},
+	body: {
+		maxWidth: "40em"
+	}
+};
+
+const ThemableAlert = props => (
+    <Alert {...props} />
+);
+
+const ThemedAlert = useSheet(styles)(ThemableAlert);
```

Comment:
why do you need the `ThemableAlert` component? why not just `useSheet` on `Alert`?